### PR TITLE
[feature] 3D icon guiding and audio system using AudioManager

### LINKED ISSUE #61: Change Icon Light Switch to 3D

### LINKED ISSUE #62: Implement Audio Manager for Interactables

### PR DESCRIPTION
**Pull request by naufalhilmi12345**
### Main changes

- Adding 3D icon to guide light switch
- Changing audio for light switch using AudioManager prefab

### Tasks List

- [close #61]
- [close #62]

### Attachments

- For 3D Icon

https://user-images.githubusercontent.com/44135006/149443928-0ee34473-8f11-4227-87d4-ebbf1512cd70.mp4

- For AudioManager
messageImage_1642128978001

### CODE REVIEW

**Review by rafiftaris — CHANGES_REQUESTED** [review]
- Change prefab name to 3d-Exclamation-Mark since it's not question mark 😆 
- Maybe put guiding icon to a directory for guiding icon themselves
- Can Audio Manager show detailed configuration options for Audio Source like when Audio Source become an component itself. Graph distance adjustment and other things may be needed to be customized

**Review by ahmadhaydar — APPROVED** [review]
:poggers:

File: Assets/Dev/Scripts/Interactables/Interactable.cs [hunk 1]
```diff
@@ -71,11 +70,11 @@ protected void PlayAudio(bool isOn)
     {
         if (isOn)
```
Comment (raihansyahdean) [inline Assets/Dev/Scripts/Interactables/Interactable.cs:71]: rename boolean to state

File: Assets/Dev/Scripts/Interactables/Interactable.cs [hunk 2]
```diff
@@ -71,11 +70,11 @@ protected void PlayAudio(bool isOn)
...
         {
-            audioOn.Play();
+            audioManager.Play("Switch_On");
```
Comment (raihansyahdean) [inline Assets/Dev/Scripts/Interactables/Interactable.cs:73]: change functionality to only audioManager.Play("name")

**Review by raihansyahdean — CHANGES_REQUESTED** [review]
Add audio functionality (if-else) in:
- LightSwitchController
- DummyFlashlight

Add AudioManager in DummyFlashlight prefab

**Review by rafiftaris — APPROVED** [review]

**Review by raihansyahdean — APPROVED** [review]

**Pull request merged**